src/allmydata/scripts/stats_gatherer.py

```diff
@@ -42,6 +46,11 @@ def create_stats_gatherer(config, out=sys.stdout, err=sys.stderr):
     f = open(os.path.join(basedir, "tahoe-stats-gatherer.tac"), "wb")
     f.write(stats_gatherer_tac)
     f.close()
+    pickledir = config.get("pickledir", "")
+    if pickledir:
```

Comment:
Uh, it took me a while to figure this out. So, pickedir is a _file_ with location to pickle directory, right?

Can it be an option in .cfg file?